observability/logging/logging-6.0/log6x-clf.adoc

```diff
@@ -88,6 +88,8 @@ syslog:: Forwards logs to an external syslog server.
 
 Each output type has its own configuration fields.
 
+include::modules/log6x-configuring-otlp-output.adoc[leveloffset=+1]
```

Comment:
Was this line and the "configuring otlp output" file added to this PR intentionally? It seems unrelated to the release notes and the pico-size PR also seems to include the same file.

Reply:
nope, its a release branch thing, it isnt actually part of this PR.

Reply:
I'm not sure I understand the answer. It looks to me as if the first commit of this PR should not be in it. Is that what you are saying?